docs/getting-started.md

```diff
@@ -17,7 +17,7 @@ dependencies:
 
 dev_dependencies:
   floor_generator: ^1.0.1
-  build_runner: ^1.12.2
+  build_runner: ^2.0.0
```

Comment:
💡 We could also update the `build_runner` versions in `README.md` and `floor/README.md`.